### PR TITLE
Update jupyter-book to 0.7.0.b2

### PR DESCRIPTION
**Pull request by joergbrech**
To Do:
 - [x] Fix broken images, especially GIFs don't seem to work properly
 - [x] Doublecheck all images
 - [x] Doublecheck all inline equations. Some look weird
 - ~~[ ] Test thebelab~~ *It's ok to do this after merge*
 - [x] Fix broken toc: The "Matlab Getting Started" sections are not referenced correctly
 - [x] Fix citations
 - [x] Doublecheck page titles
 - [x] Add link to Github Repo
 - [x] Doublecheck cells, that should not include output (notebooks are executed at build time)
 - [x] Make sure CI still works: We now need octave and the octave kernel

Here is the documentation for jupyter-book's new backend: https://jupyterbook.org/intro.html

### CODE REVIEW

**Comment by joergbrech** [comment]
Here is a preview: https://586-168147770-gh.circle-artifacts.com/0/html/content/intro.html

**Comment by MartinKliemank** [comment]
- Maths and Image is broken in both
https://592-168147770-gh.circle-artifacts.com/0/html/content/05_interp_approx/exercises_00_linear_interpolation.html 

https://592-168147770-gh.circle-artifacts.com/0/html/content/05_interp_approx/exercises_01_linear_approximation.html

 - remove outputs from cells, maths somewhat broken
https://592-168147770-gh.circle-artifacts.com/0/html/content/05_interp_approx/exercises_02_spline_interpolation.html

**Comment by joergbrech** [comment]
Danke @MartinKliemank, jetzt müsste es ok sein. Hier das aktuelle Preview: https://606-168147770-gh.circle-artifacts.com/0/html/intro.html

@MartinKliemank, @prosauer, @sandro-elsweijer: Könntet ihr bei Gelegenheit mal eure Seiten überfliegen und schauen ob noch irgendetwas nicht passt? Gibt es sonst irgendwelche Anmerkungen/Anregungen? Ich würde gerne diesen Donnerstag (14.5) mergen.

File: content/04_nichtlineare_gleichungen/exercises_00_bierschaum.ipynb [hunk 1]
```diff
@@ -542,7 +542,7 @@
    "source": [
     "### Zusätzliche Informationen\n",
     "\n",
-    "Zusätzliche Informationen zum Verfall von Bierschaum finden Sie in {% cite Theissen2009 %} und {% cite Leike2001 %}."
```
Comment (prosauer) [inline content/04_nichtlineare_gleichungen/exercises_00_bierschaum.ipynb:545]: Zeile 498 jeweils ein $ vor und hinter Gleichung zu viel.

Comment (prosauer) [reply content/04_nichtlineare_gleichungen/exercises_00_bierschaum.ipynb:545]: Auch hier sind die Outputs der Codeblöcke. Also "created file '/mtn/c/...'

File: content/08_eigenwerte/exercises_01_roots.ipynb [hunk 2]
```diff
@@ -37,7 +37,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%file z = my_roots(C)\n",
```
Comment (prosauer) [inline content/08_eigenwerte/exercises_01_roots.ipynb:40]: In exercises_00_qr ist noch ein math-Fehler in Aufgabe 1.

File: content/08_eigenwerte/exercises_03_beam_modes.ipynb [hunk 3]
```diff
@@ -14,7 +14,7 @@
     "\n",
     "## Die Modellgleichungen\n",
     "\n",
-    "Da Bild kommt uns bekannt vor! Wir können dieselben Modellgleichungen wie in [dieser Aufgabe zu linearen Gleichungssystemen](02_lineare_gleichungen/exercises_00_highline) verwenden, allerdings vernachlässigen wir die externe Kraft und die Dämpfung.\n",
```
Comment (prosauer) [inline content/08_eigenwerte/exercises_03_beam_modes.ipynb:17]: Math-Fehler in "Die Modellgleichungen"

File: content/10_bvp/exercises_00_flexible_cable.ipynb [hunk 4]
```diff
@@ -66,7 +66,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Hinweis:** In [diesem Kapitel](00_einleitung/matlab_02_scripts_and_function) wurde eine Funktion für das Bisektionsverfahren entwickelt. Verwenden Sie diese Funktion in ihrer Implementierung, um das Rad nicht neu erfinden zu müssen."
```
Comment (prosauer) [inline content/10_bvp/exercises_00_flexible_cable.ipynb:69]: Math-Fehler in Aufgabe 2

Comment (prosauer) [reply content/10_bvp/exercises_00_flexible_cable.ipynb:69]: Math-Fehler in exercises_01_nuclear_reactor Tabelle

**Comment by joergbrech** [comment]
Neues Preview: https://608-168147770-gh.circle-artifacts.com/0/html/intro.html

**Comment by joergbrech** [comment]
Can I get a preview?

**Pull request merged**